visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -139,7 +139,9 @@ void MotionPlanningFrame::computeExecuteButtonClicked()
   if (move_group_ && current_plan_)
   {
     ui_->stop_button->setEnabled(true); // enable stopping
-    bool success = move_group_->execute(*current_plan_);
+    bool success =
+      move_group_->validatePlan(*current_plan_) &&
```

Comment:
instead of chaining these, i think it would be much nicer to have a console output explaining why validatePlan failed